content/influxdb/v2.0/get-started.md

```diff
@@ -241,6 +241,18 @@ When installed as a service, InfluxDB stores data in the following locations:
 - **Key-value data:** `/var/lib/influxdb/influxd.bolt`.
 - **influx CLI configurations:** `~/.influxdbv2/configs` _(see [`influx config`](/influxdb/v2.0/reference/cli/influx/config/) for more information)_ .
 
+#### Pass arguments to systemd
+
+1. Add one or more lines like the following containing arguments for `influxd` to `/etc/default/influxdb2`:
+   ```
+   ARG1="--http-bind-address :8087"
+   ARG2="<another argument here>"
+   ```
+2. Edit the `/lib/systemd/system/influxdb.service` file as follows:
+   ```
+   ExecStart=/usr/bin/influxd $ARG1 $ARG2
+   ```
+
```

Comment:
You may add a quick note about passing custom configuration settings to the InfluxDB service. Something like:

> To customize your InfluxDB configuration, use either command line flags (arguments), environment variables, or an InfluxDB configuration file. See [InfluxDB configuration options](/influxdb/v2.0/reference/config-options/) for more information.